include/crm/stonith-ng.h

```diff
@@ -708,6 +709,11 @@ bool stonith_agent_exists(const char *agent, int timeout);
  */
 const char *stonith_action_str(const char *action);
 
+int stonith_validate(stonith_t *st, int call_options, const char *rsc_id,
```

Comment:
Let's make this internal. Eventually libstonithd will be merged into libcrmservice and get a new public API